pages/linux/logwatch.md

```diff
@@ -0,0 +1,11 @@
+# logwatch
+
+> Analyzes and summarizes system logs.
+
+- Analyze yesterday's logs in medium detail:
+
+`logwatch --range yesterday --detail med`
+
+- Get the summarized login information for the last 3 weeks:
+
+`logwatch --range 'since 3 weeks ago' --detail low --service pam_unix `
```

Comment:
Please use tokens for the 3 options. And I think, instead of specifying med and low in separate examples, maybe you can mention 'in a given detail' in the description and write '--details {{low|medium|others}}' in the example. A shorter and concise way to express more.